### PR TITLE
Update User Guide

### LINKED ISSUE #82: Update User Guide

### PR DESCRIPTION
**Pull request by nihalramesh12629**
Fixes #82

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -102,14 +102,18 @@
 
 .list-cell:filled:even {
     -fx-background-color: #3c3e3f;
+    -fx-border-color: #f2e7e6;
+    -fx-border-width: 0.5;
 }
 
 .list-cell:filled:odd {
     -fx-background-color: #515658;
+    -fx-border-color: #f2e7e6;
```
Comment (amoschee) [inline src/main/resources/view/DarkTheme.css:111]: looks good

**Review by amoschee — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -149,6 +149,36 @@ Examples:
...
+* Sorts the results based on the specified `ATTRIBUTE`. The available attributes are `name`, `subject`, `class`.
+* `sort name` sorts the results by name in alphabetical order
+* `sort subject` sorts the results by the subjects they take in alphabetical order
+* `sort class` sorts the results by class in alphanumerical order.
+* The command applies to the current list of displayed results. 
+* Sorting is case-insensitive.
+
+Examples:
+* `list` followed by `sort name` sorts all students in the address book by name.
+
```
Comment (AgentHagu) [inline docs/UserGuide.md:167]: Nice explanation of Undo with good examples!

**Review by AgentHagu — APPROVED** [review]
LGTM

**Pull request merged**